terraform/services/opt-out-import/main.tf

```diff
@@ -41,7 +41,7 @@ module "opt_out_import_function" {
   name        = local.full_name
   description = "Ingests the most recent beneficiary opt-out list from BFD"
 
-  handler = var.app == "ab2d" ? "gov.cms.ab2d.optout.OptOutHandler" : "bootstrap"
+  handler = var.app == "ab2d" ? "gov.cms.ab2d.optout.OptOutHandler" : var.app == "bcda" ? "opt-out-import" : "bootstrap"
```

Comment:
Would you mind switching this to a map as we did with `db_sg_name`, etc. in the locals? I think it makes sense to do so once each of the three APIs has its own value.